TI-reports/2025/2025-Q3-SCP-WG.md

```diff
@@ -0,0 +1,118 @@
+# 2025 Q3 Securing Critical Projects Working Group
+
+## Overview
+
+Sub-projects continue to run. Malicious Packages continues to
+attract interest, and is splitting from Package Analysis. Attendance is low.
+
+[2025-Q2 Update](2025-Q2-SCP-WG.md)
+
+## Identifying Critical Projects
+
+### Purpose
+
+Open Source Software has long suffered from a "tragedy of the commons"
+problem. Organizations large and small make use of OSS every day, but many
+projects are struggling for the time, resources and attention they need.
+
+This is a resource allocation problem - and we can help solve it together. We
+need ways to connect critical projects we all rely on with organizations that
+can provide them with support.
+
+[MVSR Link](https://github.com/ossf/wg-securing-critical-projects/blob/main/MVSR.md)
+
+### Current Status
+
+- Leveraging new, more comprehensive data sets
+    - [Census III](https://www.linuxfoundation.org/research/census-iii)
+    - [Ecosyste.ms](https://ecosyste.ms/) data
+
+### Up Next
+
+- Now that we have a good idea of critical projects, OSTIF is proposing a Managed Audit Program to go out and do the following to projects identified as critical:
+    - Establish contact with maintainer(s)/community.
+    - Collaborate with them to determine their security posture and needs.
+    - Commission security experts to assess the project, do threat modeling and perform third party review, build/augment security tooling, and review the project for supply chain security risks.
+    - Commission security experts to work with the project on bug and vulnerability fixes, tooling and supply chain security improvements. 
+    - Document and publish the work done in a report. 
+    - Publish annual impact reports of the work done (bugs/vulns fixed, tooling implemented, etc) 
+
+## Criticality Score
+
+### Purpose
+
+1. Generate a criticality score for every open source project.
+
+1. Create a list of critical projects that the open source community depends
+   on.
+
+1. Use this data to proactively improve the security posture of these critical
+   projects.
+
+### Current Status
+
+- Run not working regularly, but get a successful run here and there
+- Working through quota issues
```

Comment:
Can you please elaborate on this? What sort of quota issues are you running into?

Reply:
Updated.